cypress/integration/read-demo.spec.js

```diff
@@ -1,51 +1,268 @@
-      // TODO
+    describe('Demo 1B', () => {
+      beforeEach(() => {
+        cy.get('[data-cy=password-input-box]').type(secret)
+        cy.get('[data-cy=submit-password-btn]').click()
+        cy.get('[data-cy=hacker-app]').click()
+        cy.get('[data-cy=modal-btn-yes-aarch64]').click()
+        cy.get('[data-cy=button-side]').click()
+      })
+
+      it('confirms styling and DOM elements of Demo 1B', () => {
+        // TODO refer to demo 1a it block
+        cy.get('[data-cy=content-container]')
+          .should('have.css', 'background-color')
+          .and('contain', 'rgb(255, 255, 255)')
+        cy.get('[data-cy=password-input-box]')
+          .should('have.css', 'background-color')
+          .and('contain', 'rgb(255, 255, 255)')
+        cy.get('[data-cy=submit-password-btn]')
+          .should('have.css', 'background-color')
+          .and('contain', 'rgba(0, 0, 0, 0)')
+      })
+
+      describe('Submits password', () => {
+        beforeEach(() => {
+          cy.get('[data-cy=password-input-box]').type(secret)
+          cy.get('[data-cy=submit-password-btn]').click()
+        })
+
+        it('renders morello hacker app', () => {
+          // TODO assert that it's morello icon?
+          cy.get('[data-cy=hacker-app]').should('exist')
+          cy.get('[data-cy=hacker-app-icon]')
+            .should('exist')
+            .and('have.css', 'background-image')
+            .and('to.be.ok')
+          cy.get('[data-cy=hacker-app-icon-text]')
+            .should('have.css', 'font-family')
+            .and('contain', 'AktivGrotesk')
+        })
+
+        it('removes password input', () => {
+          cy.get('[data-cy=password-input-box]').should('not.exist')
+          cy.get('[data-cy=submit-password-btn]').should('not.exist')
+        })
+      })
+
+      describe('Opens hacker app', () => {
+        beforeEach(() => {
+          cy.get('[data-cy=password-input-box]').type(secret)
+          cy.get('[data-cy=submit-password-btn]').click()
+          cy.get('[data-cy=hacker-app]').click()
+        })
+
+        describe('Selects [no] option', () => {
+          beforeEach(() => {
+            cy.get('[data-cy=modal-btn-no-cheri]').click()
+          })
+
+          it('updates state so moddal is no longer rendered', () => {
+            cy.get('[data-cy=modal-main]').should('not.exist')
+          })
+        })
+
+        it('renders modal with yes/no option', () => {
+          cy.get('[data-cy=modal-main]').should('exist')
+          cy.get('[data-cy=modal-main-text]')
+            .should('exist')
+            .should(
+              'include.text',
+              'Would you like to break into the system and reveal the password?'
+            )
+          cy.get('[data-cy=modal-btn-yes-cheri]')
+            .should('exist')
+            .and('include.text', 'YES')
+            .and('have.css', 'font-family')
+            .and('contain', 'Arial')
```

Comment:
are the yes/no buttons in the modal supposed to have `Arial` font? I assumed that was a mistake and fixed it in https://github.com/digicatapult/morello-ui/pull/41 but might be wrong

I think this PR should merge before that font change PR so I can change it to match whichever font is correct

Reply:
To be fair I'm not sure

Reply:
Yea, it already creates conflicts, and I got another 2 ready which will have conflicts as this one did not got merged earlier, well.